cfgov/v1/preprocessed/css/molecules/text-introduction.less

```diff
@@ -0,0 +1,60 @@
+/* topdoc
+  name: Theme variables
+  family: cfgov-molecules
+  notes:
+    - "The following color and sizing variables are exposed,
+       allowing you to easily override them before compiling."
+  patterns:
+  - name: Colors
+    codenotes:
+      - |
+        @m-text-introduction_rule
+  tags:
+  - less
+*/
+@m-text-introduction_rule: @gray-50;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/4b79432ee367905b603bc84da58c75519f18404a Added rule color variable to cf-enhancements.